Ugly hack to avoid hanging in operation involving register

 -- Rafael Laboissiere <rafael@debian.org>  Thu, 13 Oct 2005 22:12:06 +0200

--- praat-4.3.24.orig/dwsys/NUMcblas.c
+++ praat-4.3.24/dwsys/NUMcblas.c
@@ -1443,6 +1443,7 @@
 	double ret_val;
 
 	ret_val = *a + *b;
+	close (open ("/dev/null", 0));
 	return ret_val;
 }								/* dlamc3_ */
 
